stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -341,6 +340,7 @@ internal class PaymentSheetViewModel internal constructor(
                 workContext = Dispatchers.IO
             )
 
+            @Suppress("UNCHECKED_CAST")
```

Comment:
Cleanup some warning, so I don't get an error on commit from android studio.

Reply:
I think it's ok to omit this annotation, I feel like they can be a bit noisy.